virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc

```diff
@@ -0,0 +1,51 @@
+[id="virt-managing-vm-snapshots"]
+= Managing virtual machine snapshots
+include::modules/virt-document-attributes.adoc[]
+:context: virt-managing-vm-snapshots
+
+toc::[]
+
+You can create and delete virtual machine (VM) snapshots for VMs, whether the VMs are powered off (offline) or on (online). You can only restore to a powered off (offline) VM. {VirtProductName} supports VM snapshots on:
```

Comment:
"on the following:"

or

"on the following providers:"

or similar, to avoid continuing the sentence into the list.